pkg/cmd/templatefuncs.go

```diff
@@ -284,6 +284,21 @@ func (c *Config) statTemplateFunc(name string) any {
 	}
 }
 
+func (c *Config) strEqFoldTemplateFunc(s1 string, s2 ...string) bool {
+	if (len(s2) == 0) {
+		panic(fmt.Errorf("expected 2 or more arguments, got %d", (1 + len(s2))))
+	}
+
+	result := false
+	for _, s := range s2 {
+		result = strings.EqualFold(s1, s)
+		if !result {
+			break
+		}
+	}
+	return result
```

Comment:
We can simplify this code with an early return:

```go
for _, s := range s2 {
    if strings.EqualFold(s1, s) {
        return true
    }
}
return false
```

Reply:
Isn't the intention to check if all provided strings match? My understanding is you would need to invert this as you would only return early if they don't at any point.

Reply:
What would be the use case for checking if all provided strings match the first argument? Surely you want to check if *any* provided strings match the first argument?

Reply:
Isn't that more of an `or` than an `eq`? We can certainly go in that direction, but the function probably needs a better name than `strEqFold`. Maybe something like `strCompareAny`?